test/0.8.9/self-owned-steth-burner.test.js

```diff
@@ -0,0 +1,698 @@
+const { assertBn, assertRevert, assertEvent, assertAmountOfEvents } = require('@aragon/contract-helpers-test/src/asserts')
+const { ZERO_ADDRESS, bn } = require('@aragon/contract-helpers-test')
+const { newDao, newApp } = require('../0.4.24/helpers/dao')
+
+const { assert } = require('chai')
+
+const SelfOwnerStETHBurner = artifacts.require('SelfOwnedStETHBurner.sol')
+
+const NodeOperatorsRegistry = artifacts.require('NodeOperatorsRegistry')
+
+const LidoMock = artifacts.require('LidoMock.sol')
+const LidoOracleMock = artifacts.require('OracleMock.sol')
+const DepositContractMock = artifacts.require('DepositContractMock.sol')
+const RewardEmulatorMock = artifacts.require('RewardEmulatorMock.sol')
+const CompositePostRebaseBeaconReceiver = artifacts.require('CompositePostRebaseBeaconReceiver.sol')
+
+const ERC20OZMock = artifacts.require('ERC20OZMock.sol')
+const ERC721OZMock = artifacts.require('ERC721OZMock.sol')
+
+const ETH = (value) => web3.utils.toWei(value + '', 'ether')
+// semantic alias
+const stETH = ETH
+
+contract('SelfOwnedStETHBurner', ([appManager, voting, deployer, depositor, anotherAccount, ...otherAccounts]) => {
+  let oracle, lido, burner
+  let treasuryAddr
+  let dao, acl, operators
+  let compositeBeaconReceiver
+
+  beforeEach('deploy lido with dao', async () => {
+    const lidoBase = await LidoMock.new({ from: deployer })
+    oracle = await LidoOracleMock.new({ from: deployer })
+    const depositContract = await DepositContractMock.new({ from: deployer })
+    const nodeOperatorsRegistryBase = await NodeOperatorsRegistry.new({ from: deployer })
+
+    ;({ dao, acl } = await newDao(appManager))
```

Comment:
Seems hacky to me mostly because of semicolon, it's correct but this syntax construction is not very common is javascript land (especially it looks weird because of semicolon in the beginning of string). Please consider to split this into several lines.

Reply:
Ok, it's not a big deal — going to update it with the next changes bunch.

Things are looking messy, ugh. We already have such code style in the `deposit.test.js` and `lido.test.js`.